src/components/Reports/FourteenMonthReports/useCsvData.test.ts

```diff
@@ -164,7 +165,7 @@ describe('useCsvData', () => {
       ],
       [
         'Quarterly Partner',
-        'Partner - Financial',
+        'PARTNER_FINANCIAL',
```

Comment:
I think we want the CSV to export the nicer-formatted version of the status. Will it be doing that?

Reply:
And then we can change this test to check that it gets translated/formatted.